translator/lib/SPIRV/SPIRVReader.cpp

```diff
@@ -557,6 +558,7 @@ class SPIRVToLLVM {
   RemappedTypeElementsMap RemappedTypeElements;
   DenseMap<Type *, bool> TypesWithPadMap;
   DenseMap<std::pair<SPIRVType*, uint32_t>, Type *> OverlappingStructTypeWorkaroundMap;
+  DenseMap<BasicBlock*, DenseMap<BasicBlock*, unsigned>> BlockToPredecessorCount;
```

Comment:
Is it possible to use struct DenseMap<std::pair<BasicBlock*, BasicBlock*>, unsiged>>, then we needn't copy DenseMap when add new keys.